src/main/java/org/folio/service/consortium/ConsortiumConfigurationService.java

```diff
@@ -67,4 +70,12 @@ private CompletableFuture<Optional<ConsortiumConfiguration>> getConsortiumConfig
       }).toCompletionStage().toCompletableFuture();
   }
 
+  public Future<RequestContext> cloneRequestContextIfNeeded(RequestContext requestContext, Location location) {
```

Comment:
can we move to ReqestContextUtils, I saw similar method. Maybe we can use as overloaded methods